### PR TITLE
Update zh-TW translation

### PR DESCRIPTION
**Pull request by tan9**
1. Add missed entry in `urls.js`.
2. Align to latest root version.
3. Refine transalations.

### CODE REVIEW

**Comment by tan9** [comment]
Travis CI failed as `cla-check-pull` task:

```
Running "cla-check-pull" task
Unexpected response from api.github.com
statusCode: 403
```

I singed the [CLA](http://dev.brackets.io/brackets-contributor-license-agreement.html) as per described in [CONTRIBUTING.md](https://github.com/adobe/brackets/blob/master/CONTRIBUTING.md). Am I missing anything?

**Comment by rroshan1** [comment]
@tan9 I just had to restart the Travis CI build after you signed the CLA. The build has all checks passed now.
Before we can merge your changes, can you tag someone for linguistic review?

**Comment by MarcelGerber** [comment]
@yiming @jenghung @lyhcode Could you please take a look at @tan9's translation?

**Comment by tan9** [comment]
Do you have something date like **Last Call for 1.5 Translations**?
Maybe we should wait until then, and find more reviewers to refine this PR.

**Comment by nethip** [comment]
@tan9 Yes we will be doing 1.5 release in a couple of weeks and that is when we will call for translations. Nevertheless, if it looks good we should push this anyway to master.

**Comment by ingorichter** [comment]
It would be great to have another pair of eyes on this PR. As @MarcelGerber already asked for help, is @yiming or @jenghung or @lyhcode available for a review? Thanks.

File: samples/zh-tw/Getting Started/index.html [hunk 1]
```diff
@@ -28,42 +27,60 @@
...
+            只要在快速編輯器的範圍外按下 <kbd>ESC</kbd> 鍵，就能關掉所有快速編輯器。
+            快速編輯也能找到定義在 LESS 及 SCSS 檔案中的規則，就算是巢狀規則也沒問題。
         </p>
         
         <samp>
             想親身體驗嗎? 把游標移到上面的 <!-- <samp> --> 標籤中，按下 <kbd>Cmd/Ctrl + E</kbd>。
-            您應該就會看到 CSS 快速編輯器出現在上方，馬上就能看到並修改套用在該標籤上的 CSS 規則。
-            快速編輯功能也支援 class 及 id 屬性喔!
+            您應該就會看到 CSS 快速編輯器出現在上方，顯示出所有套用到的 CSS 規則。
+            快速編輯功能也支援 class 及 id 屬性。搭配 LESS 或 SCSS 檔嘛會通喔。
```
Comment (jenghung) [inline samples/zh-tw/Getting Started/index.html]: some people will confuse "嘛會通喔" if they are not familiar with Taiwanese Hokkien.

**Comment by jenghung** [comment]
I reviewed @tan9 commits and most of translations is appropriate. I left some comments above. Thanks @tan9 make the zh-TW translation better!

**Comment by abose** [comment]
Thanks all. Merging for 1.5 release.

**Pull request merged**